### ISSUE TITLE
Getting started / tutorial documentation for MDL

### ISSUE THREAD

**Issue by sgomes** [google/material-design-lite#1311]
Our documentation is extensive, but not easy to grok. A set of getting started guides would go a long way in making it easier for developers to learn MDL.

**Comment by SeanBannister** [comment]
As someone new to MDL who's dug pretty deep the past month, I totally agree that the "documentation is extensive, but not easy to grok" a getting starting guide would of been great. I initially downloaded the templates to learn the basics and then read the entire component documentation before I got started. It took about a week to really feel like I could get things done.

**Comment by sgomes** [comment]
Thanks, @SeanBannister, that's very useful feedback! We definitely want to get developers up and running in much less time than that.

Out of curiosity, were there any big pain points you came across, other than layout?

**Comment by SeanBannister** [comment]
@sgomes I was certainly able to play with MDL the first day but it took that week to feel comfortable.

There were a number of pain points and I've [filled issues for everything](https://github.com/google/material-design-lite/issues?utf8=%E2%9C%93&q=author%3ASeanBannister+), most are now fixed.

I'm also trying to contribute and that was the largest hurdle I faced, trying to get the environment setup under windows took a number of attempts (most bugs now fixed), but I'm there now. Just doing a PR at the moment for a very small doc inconsistency.
